codereadyworkspaces/stack/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM registry.redhat.io/codeready-workspaces/plugin-java8-rhel8:2.10-4
+
+ENV ARGOCD_VERSION=2.0.5
+ENV YQ_VERSION=v4.11.1
+ENV HELM_VERSION=v3.6.0
```

Comment:
Use latest helm
```
ENV HELM_VERSION=v3.6.3
```

also .. $HOME does not work with this helm
```
[1001@23ede950920b tmp]$ helm version
WARNING: stat /home/jboss/.config/helm/registry.json: permission denied
```

But
```
[1001@23ede950920b tmp]$ HOME=/tmp
[1001@23ede950920b ~]$ helm version
version.BuildInfo{Version:"v3.6.3", GitCommit:"d506314abfb5d21419df8c7e7e68012379db2354", GitTreeState:"clean", GoVersion:"go1.16.5"}
```
is OK